release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+- [#4590](https://github.com/grafana/k6/pull/4590) moves a docker-compose example with InfluxDB to the `examples/docker-compose` directory and adds an opentelemetry example.
+- [#4602](https://github.com/grafana/k6/pull/4602), [#4629](https://github.com/grafana/k6/pull/4629) Better error message on options error from script. k6 now will try to print the part of the options that fails as JSON.
+
+## Bug fixes
+
+- [#4544](https://github.com/grafana/k6/pull/4544) Fix race in ReadableStream.cancel and run WPT test with race detection for easier finding of similar problems.
+- [#4567](https://github.com/grafana/k6/pull/4567), [#4558](https://github.com/grafana/k6/pull/4558), [#4574](https://github.com/grafana/k6/pull/4574), [#4610](https://github.com/grafana/k6/pull/4610) Fixes race conditions with the browser module's `ElementHandle`, `Mouse`, `Keyboard`, and `Response`.
+
+## Maintenance and internal improvements
+
+- [#4519](https://github.com/grafana/k6/pull/4519) Moves the Prometheus remote write outputs' code to the k6 repository.
+- [#4545](https://github.com/grafana/k6/pull/4545) Trying to stabilize MacOS CI by making browser throttle.js easier to run for it.
+- [#4546](https://github.com/grafana/k6/pull/4546) Make `TestConstantArrivalRateRunCorrectRate` less flaky.
+- [#4552](https://github.com/grafana/k6/pull/4552) Updates the fallback x509 certificates for CA roots.
+- [#4561](https://github.com/grafana/k6/pull/4561) Update Sobek to fix a bug in `Function.apply`.
+- [#4563](https://github.com/grafana/k6/pull/4563) Fix `k6/timers` reporting as always being used.
+- [#4562](https://github.com/grafana/k6/pull/4562) Moves the opentelemetry outputs' code to the k6 repository.
+- [#4603](https://github.com/grafana/k6/pull/4603) Homogenize package docs comment block format.
+- [#4604](https://github.com/grafana/k6/pull/4604) Bump to golangci-lint 1.64.6 - to support go 1.24.
+- [#4605](https://github.com/grafana/k6/pull/4605) Refactor the code responsible for `lib.RuntimeOptions` loading.
+- [#4606](https://github.com/grafana/k6/pull/4606) Bump k6 to depend on Go 1.23 and build with 1.24.
+- [#4611](https://github.com/grafana/k6/pull/4611) Extract the HDR histogram implementation into a shared package.
+- [#4616](https://github.com/grafana/k6/pull/4616), [#4613](https://github.com/grafana/k6/pull/4613), [#4627](https://github.com/grafana/k6/pull/4627), [#4615](https://github.com/grafana/k6/pull/4615), [#4592](https://github.com/grafana/k6/pull/4592), [#4578](https://github.com/grafana/k6/pull/4578), [#4554](https://github.com/grafana/k6/pull/4554), [#4553](https://github.com/grafana/k6/pull/4553), [#4552](https://github.com/grafana/k6/pull/4552), [#4527](https://github.com/grafana/k6/pull/4527), [#4550](https://github.com/grafana/k6/pull/4550) Update direct dependencies.
+
+## Roadmap
+
+This release marks a special, non-conventional milestone in the k6 software life-cycle, serving as a preview of the upcoming version 1.0.0.
```

Comment:
I'd suggest putting this at the top of the document. First it's align with the release highlights, secondly, but probably what's more essential, it's immediately answering the question on why after `v0.57.0` we're issuing the `v1.0.0-rc1`

Reply:
Done